apps/vr-tests-react-components/screener.config.js

```diff
@@ -17,26 +17,33 @@ function getCurrentHash() {
 
   return '';
 }
+/**
+ *
+ * @param {string} SCREENER_API_KEY
+ * @param {string} BUILD_SOURCEBRANCHNAME
+ * @param {string} DEPLOYURL
+ * @param {string} SYSTEM_PULLREQUEST_TARGETBRANCH
+ * @returns
+ */
+function getConfig(SCREENER_API_KEY, BUILD_SOURCEBRANCHNAME, DEPLOYURL, SYSTEM_PULLREQUEST_TARGETBRANCH) {
```

Comment:
```suggestion
function getConfig(screenerApiKey, sourceBranchName, deployUrl, targetBranch) {
```

These are no longer env variables but parameters, we can make their usage more readable here